aep/general/0133/create.oas.yaml

```diff
@@ -0,0 +1,67 @@
+---
+openapi: 3.0.3
+info:
+  title: Library
+  version: 1.0.0
+paths:
+  /publishers/{publisherId}/books/{bookId}:
+    parameters:
+      - name: publisherId
+        in: path
+        description: The publisher to list books for.
+        required: true
+        schema:
+          type: string
+      - name: bookId
+        in: path
+        description: The name of the book.
+        required: true
+        schema:
+          type: string
+    post:
+      operationId: createBook
+      description: Create a Book
+      requestBody:
+        description: The book that is being created.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Book'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Book'
+components:
+  schemas:
+    Book:
+      x-aep-resource:
+        singular: book
+        plural: books
+        patterns:
+          - 'publishers/{publisher}/books/{book}'
+      description: A representation of a single book.
+      type: object
+      properties:
+        name:
```

Comment:
Other examples skip the full resource definition. Since you've added the full resource, `name` be `path`? I find `path` in [AEP-122](https://aep.dev/122#fields-representing-resource-paths) and [AEP-148](https://aep.dev/148#resource-paths-and-ids) and no references to `name`.

Reply:
Same comment for the `Book` component schemas in the other examples.